### ISSUE TITLE
shield status reveals 'API Version: (development)'

### ISSUE THREAD

**Issue by bodymindarts** [starkandwayne/shield-boshrelease#41]
```
$ shield status -k                                                               
Using http://52.40.115.221 (proto) as SHIELD backend                                                                                  

Name:        acf-global- S.H.I.E.L.D                                                                                                  
API Version: (development)
```

**Comment by geofffranks** [comment]
How did you install shield?

**Comment by bodymindarts** [comment]
boshrelease v6.3.6 deployed via BOSH
shield cli v0.6.6 downloaded from github.

**Comment by jhunt** [comment]
Yeah, for this to work, we need to stop building the SHIELD software in the BOSH release, and instead use binary releases generated by the SHIELD project itself.

**Comment by jhunt** [comment]
Fixed in latest SHIELD BOSH release (v6.4.0) with SHIELD 0.7.2
